Packs/Salesforce/IncidentFields/incidentfield-Salesforce_Priority.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
@bakatzir @silviudm consider using the "Source Priority" field instead.

Reply:
fixed

Reply:
fixed

Reply:
@silviudm I don't see that you changed it to "Source Priority". Do you thing there's a reason for creating a new incident field here instead of using an existing one?